### PR TITLE
Adding ATS Link 

### PR DESCRIPTION
**Pull request by pellicceama**
My understanding of links is that they are simply configured at the pipeline level. Hence, I added an 'ats' link to the enum and implemented an `atsLink` function. 

Two callouts: 
1. I'm importing `atsLink` separately to the` bankingLink` .  Left a comment called 'Amadeo Q'
2. I noticed that lever was missing an `Offer` mapper, so I added one. However, I'm not 100% sure how this works or how to handle fields that exist in one integration but not in the other. 

I haven't tested this as I'm not in a great position to do so atm, but hopefully I can get some early feedback.

### CODE REVIEW

File: packages/engine-backend/services/sync-service.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ import {
 } from '@openint/cdk'
 import type {z} from '@openint/util'
 import {rxjs} from '@openint/util'
+// Amadeo Q: how do I make the atsLink part of the openint/cdk? is there some sort of release process?
```
Comment (tonyxiao) [inline packages/engine-backend/services/sync-service.ts:22]: it should not be part of cdk. it's a dependecy graph issue that I should sort out. `cdk` stands for connector development kit and it should be depended on BY things like connectors and unified APIs, not the other way around. Just a code smell that we need to fix unfortunately

File: unified/unified-ats/atsLink.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      )
+    }
+
+    const entityName = op.data.entityName
+    if (!categories[entityName]) {
+      console.warn(`Unsupported ATS entity type: ${entityName}`)
+      return rxjs.EMPTY
+    }
+
+    let mapper
```
Comment (tonyxiao) [inline unified/unified-ats/atsLink.ts:40]: `mapper = adapters.mappers[entityName]`

File: unified/unified-ats/package.json [hunk 3]
```diff
@@ -1,9 +1,12 @@
 {
   "name": "@openint/unified-ats",
   "dependencies": {
+    "@openint/cdk": "workspace:*",
```
Comment (tonyxiao) [inline unified/unified-ats/package.json:4]: Technically should only need `vdk` not `cdk` as `vdk` stands for `vertical development kit`

**Review by tonyxiao — COMMENTED** [review]
Definitely in the right direction. Might be missing something before it fully works, will have to try it out live to see

File: packages/engine-backend/services/sync-service.ts (hunk 1 above)
Comment (pellicceama) [reply packages/engine-backend/services/sync-service.ts:22]: i see, got confused by banking link being imported here and tried to reproduce it but its not part of the cdk package, hence the question

```
import {
  bankingLink,
  logLink,
  makeId,
  prefixConnectorNameLink,
  sync,
} from '@openint/cdk'
```

File: unified/unified-ats/package.json (hunk 3 above)
Comment (pellicceama) [reply unified/unified-ats/package.json:4]: It's just for the types in L3. This is also used by the bankingLink

import type {AnyEntityPayload, Id, Link} from '@openint/cdk'

**Pull request merged**